feature-libs/product/multi-dimensional/components/multi-dimensional-container/product-multi-dimensional-container.module.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import {
+  CmsConfig,
+  I18nModule,
+  provideDefaultConfig,
+  UrlModule,
+} from '@spartacus/core';
+import { ProductMultiDimensionalContainerComponent } from './product-multi-dimensional-container.component';
+import { RouterModule } from '@angular/router';
+import { ProductMultiDimensionalGuard } from '../guards/product-multi-dimensional.guard';
+import { MediaModule } from '@spartacus/storefront';
+import { ProductMultiDimensionalSelectorComponent } from './multi-dimensional-selector/product-multi-dimensional-selector.component';
+
+@NgModule({
+  imports: [CommonModule, RouterModule, UrlModule, I18nModule, MediaModule],
+  declarations: [
+    ProductMultiDimensionalContainerComponent,
+    ProductMultiDimensionalSelectorComponent,
+  ],
+  exports: [
+    ProductMultiDimensionalContainerComponent,
+    ProductMultiDimensionalSelectorComponent,
+  ],
+  providers: [
+    provideDefaultConfig(<CmsConfig>{
+      cmsComponents: {
+        // ProductVariantMultiDimensionalSelectorComponent
+        ProductVariantSelectorComponent: {
+          component: ProductMultiDimensionalContainerComponent,
+          guards: [ProductMultiDimensionalGuard],
+        },
+      },
+    }),
+  ],
+})
+export class ProductMultiDimensionalContainerModule {}
```

Comment:
[MEDIUM]
`ProductMultiDimensionalContainerModule` vs `ProductMultiDimensionalSelectorModule`?

Reply:
Honestly, I retained the container component from the previous implementation, which likely had its reasons. I suspect this is related to the order grid functionality.

Let's look into the future: Should this container component be responsible for deciding whether to display "variant option selectors" or the "order grid form"?

Please consider the following:

- The order grid is intended only for base products.

- Variant selectors are meant for variant products of a base product.

Reply:
Good insight. Thanks!

Should the "order grid form" really be displayed in the same place as the "variant selector"?

Variant selector is displayed in the right hand column
But the full grid view should be full-width I guess?
Therefore should it be really the same component?

Reply:
I've made changes in the other pr to rename this module to ProductMultiDimensionalSelectorModule, also removed the container module